tests/factories/cht/generate.js

```diff
@@ -4,10 +4,21 @@ const personFactory = require('@factories/cht/contacts/person');
 const deliveryFactory = require('@factories/cht/reports/delivery');
 const pregnancyFactory = require('@factories/cht/reports/pregnancy');
 const pregnancyVisitFactory = require('@factories/cht/reports/pregnancy-visit');
+const immunizationFactory = require('@factories/cht/reports/inmunization');
+const Faker = require('@faker-js/faker');
```

Comment:
```suggestion
const { faker: Faker } = require('@faker-js/faker');
```

Or 
 
```suggestion
const { faker } = require('@faker-js/faker');
```

Then we can use it like this: ` const daysAgo = faker.number.int({ min: 1, max: 10 });` instead of `Faker.faker.number.int(...);`